share/wake/lib/core/list.wake

```diff
@@ -802,3 +805,76 @@ export def groupBy (cmp: a => a => Boolean) (list: List a): List (List a) =
             (elem, head, rest), tail
         _ = (elem, Nil), acc
     foldr combo Nil sorted
+
+export def fromOption (x: Option a): List a =
+    getOrElse Nil $ omap (_, Nil) x
+
+# TODO: replace at with atT
+export def atT (i: Integer) (list: List a): List a =
```

Comment:
I'd like to nuke almost everything from here to the bottom of the diff.

The `xyzE` and `xyzT` methods are both too similar to existing functions and too abstract.